### ISSUE TITLE
[Bug]: Overrides ssh key in `~/.ssh/authorized_keys` when generating an ssh key

### ISSUE THREAD

**Issue by deadcoder0904** [coollabsio/coolify#1541]
### Description

so i'm using a vps that only allows to put an ssh key once while installing the os for the first time through their gui.

i put my public key `~/.ssh/id_rsa.pub` in the server's authorized keys `~/.ssh/authorized_keys` & i got access to the machine.

but then i installed coolify. when onboarding, i put my already existing ssh key but i got an error for some reason. maybe bcz i didn't put the last parts of my ssh key which is just `myname@desktopname` which i learned isn't needed at all. maybe that's why i got the error idk.

but then i thought maybe i'll let coolify generate an ssh key & it went successfully. however, i wasn't able to ssh into my remote server anymore. maybe coolify overrided the ssh key in `authorized_keys` instead of appending?

all i know is it worked before installing coolify onboarding.

### Minimal Reproduction (if possible, example repository)

n/a

### Exception or Error

n/a

### Version

n/a

**Comment by guestisp** [comment]
i don't think so, the key seems to be appended
https://github.com/coollabsio/coolify/blob/6b751f965bb6deab3423b74511451089e06f3d37/scripts/install.sh#L174

**Comment by deadcoder0904** [comment]
yeah, i did see it was indeed appended once i reinstalled another os.

im curious why the onboarding gave me error of public key denied when i entered my private key? i had to skip the onboarding flow to get it working.

if i generate ssh key during onboarding, it stopped working for some reason tho. idk why?

**Comment by deadcoder0904** [comment]
i understood that i need 2 ssh keys:

one for accessing server from my local computer & another one for coolify to access it.

there's an issue somewhere but i'm unable to pin-point it as i dont really understand devops that much so i'll close this one for now.

hopefully, i never face such issue again.
